src/Sarif.Multitool/ValidateCommand.cs

```diff
@@ -60,6 +60,8 @@ protected override SarifValidationContext CreateContext(ValidateOptions options,
 
             if (!string.IsNullOrEmpty(sarifText))
             {
+                // DISCUSS IN PR: Deserializing the object here injects "$Schema" property, corrupting our test for
+                // SARIF2008_ProvideSchema
```

Comment:
Larry - need to discuss how to solve this.

Reply:
This is also a possible bug in our Validate command. Isnt it? If you confirm, i can log a bug, mark the test cases as ignore and move on for now.

---
In reply to: [443040878](https://github.com/microsoft/sarif-sdk/pull/1921#discussion_r443040878) [](ancestors = 443040878)

Reply:
Yes, change to SARIF2008 is the right fix.

---
In reply to: [443046418](https://github.com/microsoft/sarif-sdk/pull/1921#discussion_r443046418) [](ancestors = 443046418,443040878)